### PR TITLE
Updating readme file

### PR DESCRIPTION
**Pull request by Rahulk-Rai13**
I just added another headings

### CODE REVIEW

**Comment by harinderpince** [comment]
verified you may proceed

**Comment by Rahulk-Rai13** [comment]
How dare you to touch this file

File: README.md [hunk 1]
```diff
@@ -1,2 +1,3 @@
 # learnning-git
 This i smy first repository
+# Chapter 2
```
Comment (Rahulk-Rai13) [inline README.md:3]: I never encourage to change these coments

**Comment by muditzzz** [comment]
This is it.

**Comment by Debaditya-89** [comment]
Good job

**Comment by gbulchandani** [comment]
Namaste

**Pull request merged**